packages/mobile/src/account/LinkBankAccountScreen.tsx

```diff
@@ -146,7 +156,30 @@ export function StepOne() {
     // Add a bit of a delay so that Persona can popup before switching to the spinner view
     setTimeout(() => setIsInPersonaFlow(true), 500)
   }
-  const onSuccessPersona = () => {
+
+  const isUserRegionSupported = (address: InquiryAttributes['address']) => {
```

Comment:
if this were taken outside of the StepOne function, it would be easier to unit test